src/pages/experiments/[experimentId]/plots-and-tables/frequency/index.jsx

```diff
@@ -38,7 +39,7 @@ const route = {
 const frequencyPlot = ({ experimentId }) => {
   const dispatch = useDispatch();
   const config = useSelector((state) => state.componentConfig[plotUuid]?.config);
-  const cellSets = useSelector((state) => state.cellSets);
+  const cellSets = useSelector(getCellSets());
```

Comment:
`getCellOptions` looks like another candidate for a selector perhaps

Reply:
getCellSets sounds much more straightforward as we use cellSets everywhere so it wont be ambiguous what is meant by that in my opinion.

Reply:
if we want to avoid ambiguity we could use a better name, like `getCellSetsOfType` or perhaps even better `...OfTypes` and fetch any that match one of the required types)

Reply:
but other than the weird name, do you think any other ambiguity would be introduced by moving this into a selector?